.env.example

```
@@ -0,0 +1,142 @@
+HUGGINGFACE_SUMMARY_MODEL=facebook/bart-large-cnn
+
+# Python LLM Service (Alternative to Hugging Face)
+PYTHON_LLM_URL=http://localhost:8000
+
+# =============================================================================
+
+# DOCUMENT PROCESSING
+# =============================================================================
+
+# File Upload Limits
+MAX_UPLOAD_BYTES=10485760  # 10MB
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)